includes/definitions/bti800.yaml

```diff
@@ -0,0 +1,14 @@
+os: bti800
+text: 'BTI SA-800'
+type: network
+icon: bti
+group: bti
+ifname: true
+over:
+   - { graph: device_bits, text: Traffic }
+   - { graph: device_temperature, text: Temperature (Celcius) }
+   - { graph: device_voltage, text: Voltage }
+discovery:
+    - sysObjectID:
+        - .1.3.6.1.4.1.30005.1.7
+mib-dir: bti800
```

Comment:
Rename the folder to bti

Reply:
Ok. Will do @Jellyfrog. Thanks for your feedback.